src/quo2/components/wallet/keypair/view.cljs

```diff
@@ -0,0 +1,82 @@
+(ns quo2.components.wallet.keypair.view
+  (:require
+    [quo2.components.avatars.icon-avatar :as icon-avatar]
+    [quo2.components.avatars.user-avatar.view :as user-avatar]
+    [quo2.components.icon :as icon]
+    [quo2.components.list-items.account-list-card.view :as account-list-card]
+    [quo2.components.markdown.text :as text]
+    [quo2.foundations.colors :as colors]
+    [quo2.theme :as quo.theme]
+    [react-native.core :as rn]
+    [quo2.components.selectors.selectors.view :as selectors]
+    [utils.i18n :as i18n]
+    [quo2.components.wallet.keypair.style :as style]))
+
+(defn keypair-string
+  [full-name]
+  (let [first-name (first (clojure.string/split full-name #" "))]
+    (str first-name "'s default keypair")))
```

Comment:
Does "'s default keypair" need to be imported from i18n?

Reply:
yeah this looks like something that should be translated 👍